website/versioned_docs/version-5.x/separating-applications.md

```diff
@@ -57,6 +61,7 @@ Changing which JavaScript file is loaded for each child application can be done
 
 |   | Separate code repositories possible | Independent CI builds | Separate deployments | Examples |
 | - | ----------------------------------- | --------------- | -------------------- | -------- |
-| Monorepo | :x: | :x: | :x: | &mdash; |
+| Monolith | :x: | :x: | :x: | &mdash; |
```

Comment:
Since there is no concrete definition of Monolith, I would prefer not having this row.

Reply:
Well, you had a section on this page called `Monolith` so I thought the table should be aligned with the sections.

Reply:
I don't see the section called "Monolith". Am looking at https://github.com/single-spa/single-spa.js.org/blob/master/website/versioned_docs/version-5.x/separating-applications.md

Reply:
Let's remove this row

Reply:
```suggestion
```